react/ProductSKUNames.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { useCssHandles } from 'vtex.css-handles'
+import useProduct from 'vtex.product-context/useProduct'
+
+const CSS_HANDLES = [
+  'productSKUNamesContainer',
+  'skuLineClass',
+  'skuNameClass',
+  'skuValueClass',
+] as const
+
+const getVariationsFromSelectedItem = (
+  productContext: ReturnType<typeof useProduct>
+) => {
+  if (
+    !productContext ||
+    !productContext.product ||
+    !productContext.skuSelector ||
+    !productContext.selectedItem
```

Comment:
```suggestion
    !productContext?.product ||
    !productContext?.skuSelector ||
    !productContext?.selectedItem
```

Reply:
@victorhmp In this case, I don't think it makes much of a difference